features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -372,6 +373,21 @@ internal open class RumViewScope(
         }
     }
 
+    @WorkerThread
+    private fun onStopAction(event: RumRawEvent.StopAction, writer: DataWriter<Any>) {
+        delegateEventToChildren(event, writer)
+        if (stopped) return
+        if (event.type != null) {
+            interactionToNextViewMetricResolver.onActionSent(
+                InternalInteractionContext(
+                    viewId,
+                    event.type,
+                    event.eventTime.nanoTime
```

Comment:
clarification/ The `interactionToNextViewMetricResolver` is updated from both `onStopAction()` and `onStartAction()`, and we’re using event.eventTime in both cases. This gives the impression that we're treating "action start" and "action end" interchangeably, depending on the action type 🤔.

What we need is consistency in ITNV computation, specifically in determining the T1 moment in the following diagram. It should always represent the end of the action, regardless of the action type (tap, swipe, back, click).

<img width="60%" alt="Screenshot 2024-12-04 at 09 26 02" src="https://github.com/user-attachments/assets/7e762042-4d1c-4639-aa70-252e441a0cf1">

Reply:
yes the confusion comes from the fact that we are using `StartAction` with `waitForStop:false` to send `TAP` events.

Reply:
Got it. So this is equal to "stop action" 👍.

Reply:
yes for `Tap` events this is how we report it directly through the `StartAction`. I admit it is confusing.